c_check

```diff
@@ -293,6 +293,21 @@ if (($architecture eq "riscv64")) {
     }
 }
 
+$loongarch_cc_flavor = "";
+if (($architecture eq "loongarch64")) {
+    # Currently there are only 2 flavors: new-world and old-world.
+    # They differ in ABI names accepted, so check it to avoid creating temp
+    # files.
+    $args = " -mabi=lp64d -E -o - /dev/null";
+    my @cmd = ("$compiler_name $flags $args >/dev/null 2>/dev/null");
+    system(@cmd) == 0;
+    if ($? != 0) {
+        $loongarch_cc_flavor = "OLD";
+    } else {
+        $loongarch_cc_flavor = "NEW";
+    }
+}
+
```

Comment:
As '-mabi=lp64' is only supported by old binutils (2.38) and not supported by compiler gcc. 
This check may never set flavor to "OLD" unless someone is using a transitional version compiler inside Loongson.
I suggest changing it directly to '-mabi=lp64d' to avoid confusion.

Reply:
So the LoongArch gcc provided by Deepin/UOS/Loongnix *all* support `-mabi=lp64d` now?